File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.java
class Solution {
    public int firstUniqChar(String s) {
       int[] field = new int[26];
       for(char ch: s.toCharArray()){
        field[ch-'a']+=1;
       }
       for(int i=0; i<s.length(); i++){
        char ch = s.charAt(i);
        if(field[ch-'a']==1){
            return i;
        }
       }
       return -1;
    }
}

File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        field=[0]*26
        for ch in s:
            field[ord(ch)-ord('a')]+=1
        for i,ch in enumerate(s):
            if(field[ord(ch)-ord('a')]==1):
                return i
        return -1

        
        
        